packages/react-devtools-shared/src/backend/fiber/renderer.js

```diff
@@ -155,6 +155,7 @@ type FiberInstance = {
   componentStack: null | string,
   errors: null | Map<string, number>, // error messages and count
   warnings: null | Map<string, number>, // warning messages and count
+  treeBaseDuration: number, // the profiled time of the last render of this subtree
```

Comment:
This is one unnecessary field when isProfilingSupported isn't true (i.e. in production only builds). We could potentially make that conditionally added based on build but doesn't seem worth it. It's not worse otherwise since we added a map entry for everything otherwise.